composer.json

```diff
@@ -21,8 +21,8 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "composer-plugin-api": "^2.2",
-        "php-tuf/php-tuf": "dev-main",
+        "composer-plugin-api": "^2.6",
```

Comment:
Versions before this are not supported. Automatic Updates' Package Manager also requires this.